packages/icon/src/vaadin-icon.js

```diff
@@ -68,6 +79,14 @@ class Icon extends ThemableMixin(ElementMixin(ControllerMixin(PolymerElement)))
           width: 24px;
           height: 24px;
           fill: currentColor;
+          container-type: size;
+        }
+
+        :host::after,
+        :host::before {
+          line-height: 1;
+          font-size: var(--_vaadin-font-icon-size, 100cqh);
+          position: absolute;
```

Comment:
This can be removed in the upcoming PRs in favor of hiding the internal SVG when a font icon is set using a proper API.